src/snmalloc/mem/globalalloc.h

```diff
@@ -87,6 +87,10 @@ namespace snmalloc
       }
     }
 
+    if (
+      result == nullptr && RemoteDeallocCache::remote_inflight.get_curr() != 0)
```

Comment:
Something has happened (TM) with the syntax there.  Can this be a `SNMALLOC_ASSERT_MSG`?